virus.py

```diff
@@ -0,0 +1,101 @@
+import datajoint as dj
+
+schema = dj.schema('virus', locals())
```

Comment:
let's use the updated syntax where you can omit the `locals()`. So it can now read:

```python
schema = dj.schema('virus')
```